packages/fxa-settings/src/pages/Signin/index.test.tsx

```diff
@@ -357,6 +357,18 @@ describe('Signin', () => {
                 '/connect_another_device?showSuccessMessage=true'
               );
             });
+            it('is not sent if user has 2FA enabled', async () => {
+              const beginSigninHandler = jest.fn().mockReturnValueOnce(
+                createBeginSigninResponse({
+                  verified: false,
+                  verificationMethod: VerificationMethods.TOTP_2FA,
+                })
+              );
+              const integration = createMockSigninSyncIntegration();
+              render({ beginSigninHandler, integration });
+              enterPasswordAndSubmit();
+              expect(fxaLoginSpy).not.toHaveBeenCalled();
+            });
```

Comment:
I will file an issue to move some tests over and into a new test file for this utils file, we've kind of got `handleNavigation` stuff scattered about in our other tests (likely my bad).